tests/e2e/helpers.ts

```diff
@@ -61,12 +60,18 @@ export async function archiveExistingWPs({ page }) {
     .selectOption("Puerto Rico");
 
   await page.getByLabel("MFP Work Plan").click();
+
   await page.getByRole("button", { name: "Go to Report Dashboard" }).click();
+
   const reportsPromise = page.waitForResponse(
     (response) =>
       response.url().includes("/reports/WP/PR") && response.status() == 200
   );
   await reportsPromise;
-  await archiveReports(page.getByRole("button", { name: "Archive" }), page);
+  await page.waitForTimeout(1000);
```

Comment:
we finally had to add a timeout, huh? Was it not loading reliably?

Reply:
Yeah, the check for an archive button was happening before the dashboard loaded. But maybe with some re-arranging this will get fixed.